File: app/src/main/java/mb00/android/codehub/ui/repo/inject/RepoFileActivityComponent.java
package mb00.android.codehub.ui.repo.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.repo.view.RepoFileActivity;


@Subcomponent(modules = RepoFileActivityModule.class)
public interface RepoFileActivityComponent extends AndroidInjector<RepoFileActivity> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<RepoFileActivity> {}

}

File: app/src/main/java/mb00/android/codehub/ui/repo/inject/RepoFileActivityComponent.kt
package mb00.android.codehub.ui.repo.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.repo.view.RepoFileActivity


@Subcomponent(modules = [
    RepoFileActivityModule::class
])
interface RepoFileActivityComponent : AndroidInjector<RepoFileActivity> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<RepoFileActivity>()

}
